src/components/dapp-staking/register/Community.vue

```diff
@@ -79,7 +79,7 @@ export default defineComponent({
       {
         type: CommunityType.Discord,
         handle: '',
-        iconUrl: require('src/assets/img/twitter.png'),
+        iconUrl: require('src/assets/img/discord.png'),
```

Comment:
Can we use  <astar-icon-discord /> ?

Reply:
I will think about how to use icons from Astar UI in this context. In this component we are rendering UI from the configuration defined in code and all icons are handled the same way. Most likely adding Astar UI icons will required component refactoring. I will create a ticket for that.